search/tests/test_search_view.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for repostitory listing view search."""
+
+from django.core.urlresolvers import reverse
+
+from learningresources.api import create_repo
+from learningresources.tests.base import LoreTestCase
+
+
+class TestSearchView(LoreTestCase):
+    """Test Repository listing view.."""
+    def test_single_repo(self):
+        """
+        Search should only return records for one repository
+        at a time.
+        """
+        resp = self.client.get(reverse("repositories", args=(self.repo.slug,)))
+        body = resp.content.decode('utf-8')
+        self.assertTrue(self.resource.title in body)
+
+        # Make a new repo with no content.
+        new_repo = create_repo("other", "whatever", self.user.id)
+
+        resp = self.client.get(reverse("repositories", args=(new_repo.slug,)))
+        body = resp.content.decode('utf-8')
+        self.assertFalse(self.resource.title in body)
```

Comment:
Please use the nice Django TestCase self.AssertNotContains: https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.SimpleTestCase.assertNotContains to also vet the status while you are at it.